modules/core/src/smithy4s/Service.scala

```diff
@@ -34,33 +34,44 @@ import kinds._
   *   around makes it drastically easier to implement logic generically, without involving
   *   metaprogramming.
   */
-trait Service[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _]] extends FunctorK5[Alg] with Service.Provider[Alg, Op] {
-  implicit val serviceInstance: Service[Alg, Op] = this
-  val service = this
+trait Service[Alg[_[_, _, _, _, _]]] extends FunctorK5[Alg] with Service.Provider[Alg] {
+  type Operation[E, I, O, SI, SO]
```

Comment:
allows for a long name :D (not that it was not allowed of course, just that the long version in the type parameter as just so annoying